webpack.config.js

```diff
@@ -18,6 +17,9 @@ module.exports = async (env, options) => {
     resolve: {
       extensions: [".ts", ".tsx", ".html", ".js"]
     },
+    node: {
```

Comment:
can this be removed?

Reply:
this is still needed to address compiler error